PathGenerator/PathGeneratorModel.cpp

```diff
@@ -0,0 +1,84 @@
+#include "PathGenerator.hpp"
+#include <iostream>
+#include <cmath>
+
+namespace Example
+{
+void PathGenerator::operator()(const halp::tick_flicks& t)
+{
+  // Linear path
+  if (inputs.Path == 0) {
+    std::cout << (int)t.relative_position << std::endl;
+
+    for (size_t v = 0; v < outputs.OutTab.value.size(); v++) {
+      ossia::vec2f start = inputs.pos.value[v].get<std::vector<ossia::value>>()[0].get<ossia::vec2f>();
+      ossia::vec2f end = inputs.pos.value[v].get<std::vector<ossia::value>>()[1].get<ossia::vec2f>();
+
+      start[1] = 1 - start[1];
+      end[1] = 1 - end[1];
+
+      float relativePos = fmod(t.relative_position * inputs.speed, 1.0f);
+      if (inputs.loop) {
+        if ((int)fmod(t.relative_position * inputs.speed, 2.0f) == 0) {
```

Comment:
I'd recommend taking a reference to 

     outputs.OutTab.value[v].get<ossia::vec2f>()
     
e.g.: 

     auto& out = outputs.OutTab.value[v].get<ossia::vec2f>();
     
as every call eats a bit of performance.